include/foonathan/memory/std_allocator.hpp

```diff
@@ -74,7 +74,12 @@ namespace foonathan
         /// synchronization is done via a \c Mutex which defaults to \ref default_mutex.
         /// \ingroup memory adapter
         template <typename T, class RawAllocator, class Mutex = default_mutex>
-        class std_allocator : FOONATHAN_EBO(allocator_reference<RawAllocator, Mutex>)
+        class std_allocator :
+#if defined _MSC_VER && !defined __clang__
```

Comment:
The logic seems incorrect, wouldn't you also want this for non MSVC compilers?

Reply:
That's embarrassing. You're absolutely right. My initial formulation was different, and I messed it up.